gtd/gtd-plans.js

```diff
@@ -0,0 +1,142 @@
+// GTD plans fetch
+
+function extractProductDetails(text) {
+  const internetPrefix = "INTERNET FIBRA ÓPTICA";
```

Comment:
Both of these can be outside `extractProductDetails`, instead of redefining them on every function call.